src/sentry/rules/conditions/event_frequency.py

```diff
@@ -49,20 +54,20 @@ class EventFrequencyForm(forms.Form):
     )
     value = forms.IntegerField(widget=forms.TextInput())
     comparisonType = forms.ChoiceField(
-        choices=list(sorted(comparison_types.items(), key=lambda item: item[1])),
+        choices=list(sorted(comparison_types.items(), key=lambda item: item[1])),  # type: ignore
         required=False,
     )
     comparisonInterval = forms.ChoiceField(
         choices=[
             (key, label)
-            for key, (label, duration) in sorted(
+            for key, (label, duration_) in sorted(
```

Comment:
Why this change?

Reply:
The variable is misbehaving because it's also defined inside `interval`. Since they're both unread I could rename both to `_`.

Reply:
Ah I see. Yeah if they are unused I would use `_`